### ISSUE TITLE
Allow using LambdaCase in case splitting plugin

### ISSUE THREAD

**Issue by expipiplus1** [haskell/haskell-language-server#394]
It would be nice if case splitting were configurable to generate `LambdaCase` splits. This could either be a static configuration option, or an additional suggestion in every split.

For the example here: https://github.com/haskell/haskell-language-server/pull/391#issue-483151504

```haskell
fmapEither = \fab -> \case
  Left c -> Left _ 
  Right a -> Right _
```

**Comment by isovector** [comment]
I plan to tackle this.

**Comment by jneira** [comment]
This deserves a special label :wink:

**Comment by isovector** [comment]
Implemented in 84d47cda8c31e66c461348397390505e6526cbdd; this will be fixed when we merge #431

**Comment by expipiplus1** [comment]
Thanks!
